<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-76M5JGF" failures="1" tests="4" name="Default test" time="60.926" errors="0" timestamp="2024-05-09T17:06:24 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.899"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.869">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/v1/inventory.html] but found [https://www.saucedemo.com/v1/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/v1/inventory.html] but found [https://www.saucedemo.com/v1/]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at steps.LoginSteps.userShouldMoveToHomePage(LoginSteps.java:47)
at âœ½.user should move to home page(file:///C:/Users/WELCOME/eclipse-workspace/TestingDemoProject/src/test/java/Features/demo.feature:37)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.212"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.777"/>
</testsuite> <!-- Default test -->
